python/dgl/graphbolt/impl/fused_csc_sampling_graph.py

```diff
@@ -811,7 +810,6 @@ def _sample_neighbors(
         fanouts: torch.Tensor,
         replace: bool = False,
         probs_or_mask: Optional[torch.Tensor] = None,
-        return_eids: bool = False,
```

Comment:
after this change, do we always expect `subgraph.original_edge_ids` is available? for the latest master branch code.

Reply:
The meaning of original_edge_ids has been changed

Reply:
This argument is removed, you can consider it is True for all cases now.